Dockerfile

```diff
@@ -2,10 +2,14 @@ FROM ventx/alpine:3.6
 
 ENV AWSCLI 1.11.115
 
-RUN  apk --update add git openssh-client python py-pip && \
+RUN  apk --update add git openssh-client python py-pip groff && \
   pip install --upgrade pip && \
   pip install awscli==${AWSCLI}  
+
+
+RUN echo "source ~/.aws/.bash_aws" >> /root/.bashrc
   
 WORKDIR /work
 
-ENTRYPOINT ["aws"]
+
+ENTRYPOINT ["/bin/bash"]
```

Comment:
This will break my scripts cause the idea is to run the container with aws and execute with parameter and stop the container